dwave/optimization/src/nodes/_state.hpp

```diff
@@ -147,7 +148,7 @@ class ArrayStateData {
 
     // Set the value at index, tracking the change in the diff.
     // If allow_emplace is true, do an emplace_back iff the index is equal to the current size.
-    bool set(ssize_t i, double value, bool allow_emplace = false) {
+    bool inline set(ssize_t i, double value, bool allow_emplace = false) {
```

Comment:
all methods declared in the class body are automatically marked `inline`
```suggestion
    bool set(ssize_t i, double value, bool allow_emplace = false) {
```

Reply:
TIL